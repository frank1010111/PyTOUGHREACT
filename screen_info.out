 
      ----------- TOUGHREACT V3.60demo -------
           Lawrence Berkeley National Laboratory   
                    Sep 21, 2018
 
     ***** OpenMP Parallel Code Settings **** 
    *** maximum number of threads allotted  =   8
    *** maximum number of threads available =   8
 
    --> reading multiphase flow input (flow.inp)
    --> reading solute transport input (solute.inp)
    --> reading chemical input (chemical.inp)
    --> read input data complete
 
    --> performing simulation

    END OF TOUGHREACT V3.60demo SIMULATION

    ELAPSED TIME =      0.544 SEC   CALCULATION TIME =      0.497 SEC   DATA INPUT TIME =    0.047 SEC

