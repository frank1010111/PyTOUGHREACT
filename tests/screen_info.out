 
      ----------- TOUGHREACT V3.60demo -------
           Lawrence Berkeley National Laboratory   
                    Sep 21, 2018
 
     ***** OpenMP Parallel Code Settings **** 
    *** maximum number of threads allotted  =  56
    *** maximum number of threads available =  56
 
    --> reading multiphase flow input (flow.inp)
    --> reading solute transport input (solute.inp)
    --> reading chemical input (chemical.inp)
    --> read input data complete
 
    --> performing simulation

    END OF TOUGHREACT V3.60demo SIMULATION

    ELAPSED TIME =      2.182 SEC   CALCULATION TIME =      2.166 SEC   DATA INPUT TIME =    0.016 SEC

